core/contractsdk/java/src/main/java/com/baidu/xuper/Context.java

```diff
@@ -1,18 +1,38 @@
 package com.baidu.xuper;
 
+import java.util.HashMap;
+import java.util.ArrayList;
 import java.util.Map;
 
+import com.baidu.xuper.contractpb.Contract;
+
 public interface Context {
     public Map<String, byte[]> args();
 
     public String caller();
 
+    public ArrayList<String> authRequire();
```

Comment:
It is better to use the general List type for return value.

Reply:
fixed